src/pages/contact/index.tsx

```diff
@@ -0,0 +1,32 @@
+import DefaultLayout from '@atb/layouts/default';
```

Comment:
We need to check if the env variable for the contact page is set, or else return 404 not found